matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/model/OlmInboundGroupSessionWrapper2.kt

```diff
@@ -43,6 +43,10 @@ internal class OlmInboundGroupSessionWrapper2 : Serializable {
     // Devices which forwarded this session to us (normally empty).
     var forwardingCurve25519KeyChain: List<String>? = ArrayList()
 
+    // Flag that indicates whether or not the current inboundSession will be shared to
+    // invited users to decrypt past messages
+    var sharedHistory: Boolean = false
```

Comment:
cc @bmarty does dokka support comments `//` or should we start using javadoc `/**` instead?

Reply:
Dokka only understand `/**`. I have added a note about that for the coming weekly sync.
But note the `Dokka` is not generating doc for internal classes, so not a big deal here.